packages/components/inputs/money-input/src/money-input.tsx

```diff
@@ -825,6 +832,11 @@ const MoneyInput = (props: TMoneyInputProps) => {
                 css`
                   padding-right: ${designTokens.spacing40};
```

Comment:
It seems to me the high precision badge styles should also be updated:
<img width="334" alt="Screenshot 2023-02-06 at 12 52 07" src="https://user-images.githubusercontent.com/49066275/216965324-201033b3-bda2-4bcb-83d5-78feb9e45085.png">
to match the designs
<img width="374" alt="Screenshot 2023-02-06 at 12 56 45" src="https://user-images.githubusercontent.com/49066275/216965483-6f45a3aa-10f1-4fbf-b66c-0ac95631a5dc.png">

Reply:
Updated [11712c6](https://github.com/commercetools/ui-kit/pull/2397/commits/11712c63c56868b44f14f3e3f2e0bfd229f7aa4f)